lib/ansible/plugins/inventory/onesixtyone.py

```diff
@@ -74,12 +74,13 @@
 try:
     import gevent
     from gevent import socket
+    from gevent.exceptions import LoopExit
     HAS_GEVENT = True
 except Exception:
     HAS_GEVENT = False
 
 
-class InventoryModule(BaseInventoryPlugin, Constructable, Cacheable):
+class InventoryModule(BaseInventoryPlugin):
```

Comment:
i would advise to keep constructable and just implement the functionality, it allows the user to create groups/variables via  compose so you don't have to hardcode it into the plugin

Reply:
@bcoca it *cannot* be `constructable` because i am only able to `group_by` `platform`....

Reply:
not sure i understand that, won't users also want to group by other data?

Reply:
Nope, the only relevant data is the one i am using, which is the platform itself, so additional interaction to grab more details from the device is possible through `<platform>_facts`, and grouping them with `group_by`.

Reply:
That Requries a 2nd interaction and playbook hoops, some of the data is already available as output from onesixtyone, why not make it available to user instead of immediately discarding it?